appinfo/routes.php

```diff
@@ -46,25 +46,13 @@
 	],
 	'ocs' => [
 		[
-			'name' => 'api#getRooms',
+			'name' => 'Call#getRooms',
```

Comment:
This method is in the room controller, not the call controller:
https://github.com/nextcloud/spreed/blob/5cbc34ac74b878ee83ad8bbe6c9b2d947cd23127/lib/Controller/RoomController.php#L102